lfs/lfs.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
-const (
-	Version = "1.5.0"
```

Comment:
FWIW I think it should _eventually_ in `./git-lfs.go` or something:

```go
import gitlfs "github.com/git-lfs/git-lfs"
import "fmt"

func main() {
  fmt.Println(gitlfs.Version)
}
```

Oh god, I just realized `git-lfs` is not a valid go package. What have we donnnnnne.

As far as this PR is concerned, `config.Version` is just fiiiine :+1: